pkg/workflows/gen/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"log"
+	"text/template"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/codegen"
+)
+
+//go:embed compute.go.templ
+var computeGo string
```

Comment:
Would you mind writing down instructions on how to use this generator? (either in a comment here or in a README file). This file embeds compute.go.templ but the comment on top of compute.go.templ says it was generated by this script..? I'm confused.

Reply:
The script has that line so that the generated code will all have it.  I did that for the other code generators templates as well.